src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.cs

```diff
@@ -694,6 +694,26 @@ public TextAlignment TextAlignment
 
 		#endregion
 
+		public string SelectedText
+		{
+			get => ((string)this.GetValue(TextProperty)).Substring(SelectionStart, SelectionLength);
```

Comment:
What happens if SelectionStart and SelectionLengh are out of bounds?

Reply:
if negative value an exception is triggered (cannot be negative).
![image](https://user-images.githubusercontent.com/790876/157790124-48f16b44-e99f-44ea-9ac2-3a4ed10b9cd5.png)
![image](https://user-images.githubusercontent.com/790876/157790321-0c281fd0-8b88-4c62-9840-03c8042f1c24.png)


if SelectionStart > Text.Length => Text.Length
if SelectionLength > Text.Length => 0
![image](https://user-images.githubusercontent.com/790876/157789931-062c92de-5583-467b-ac75-69873cfda9b2.png)